internal/converter/internal/staticconvert/testdata-v2_windows/integrations_v2.alloy

```diff
@@ -1,6 +1,6 @@
 prometheus.remote_write "metrics_default" {
 	endpoint {
-		name = "default-149bbd"
+		name = "default-c3a75f"
```

Comment:
This must be due to the recent Prometheus upgrade. I think the hashing function in Prometheus changed, and I had to change a lot of such test files. But I didn't change this windows-specific one.